packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -582,6 +582,38 @@ OPTIONS added to them.
 See [#906](https://github.com/aws/aws-cdk/issues/906) for a list of CORS
 features which are not yet supported.
 
+### Endpoint Configuration
+API gateway allows us to specify an [API Endpoint Type](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-api-endpoint-types.html). To define an endpoint type for the API gateway, use `endpointConfiguration` property:
```

Comment:
- Line wrap to 120 or 150

- Keep the language consistent: "API Gateway allows *you* to specify..."